utils/src/main/scala/za/co/absa/enceladus/utils/transformations/ArrayTransformations.scala

```diff
@@ -24,30 +24,31 @@ import org.apache.spark.sql.api.java.UDF2
 import org.slf4j.LoggerFactory
 import org.apache.spark.storage.StorageLevel
 
-import scala.collection.mutable.ListBuffer
-
 object ArrayTransformations {
 
-  val logger = LoggerFactory.getLogger(this.getClass)
+  private val logger = LoggerFactory.getLogger(this.getClass)
   private[enceladus] val arraySizeCols = new scala.collection.mutable.HashMap[String, String]()
 
   private val zipWithOrderUDF1 = new UDF1[Seq[Row], Seq[(Int, Row)]] {
-    override def call(t1: Seq[Row]) = {
-      if (t1 == null) null
+    override def call(t1: Seq[Row]): Seq[(Int, Row)] = {
+      // scalastyle:off if.brace
```

Comment:
I would rather see one ignore null for the method and add braces then do this.

Reply:
OK, if.brace exception will go away. I am all for it :-)